### PR TITLE
CSHARP-5266: Switch to Ubuntu for package creation

### PR DESCRIPTION
**Pull request by sanych-sun**

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -4,8 +4,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
-    <TargetFrameworks Condition="'$(IsWindows)'=='true'">$(TargetFrameworks);net472</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net472</TargetFrameworks>
```
Comment (BorisDog) [inline src/Directory.Build.props:7]: same change for tests?

**Review by BorisDog — CHANGES_REQUESTED** [review]

File: src/Directory.Build.props (hunk 1 above)
Comment (sanych-sun) [reply src/Directory.Build.props:7]: Done

Comment (sanych-sun) [reply src/Directory.Build.props:7]: Changes to tests were reverted. We need that condition there, so tests would not failure locally on non-Windows platforms.

**Review by BorisDog — APPROVED** [review]
LGTM! Please double check that the failures are not related to the changes.

**Pull request merged**